chapters/queues.md

```diff
@@ -0,0 +1,100 @@
+# Queues
+
+> Further resources for queues from [AMD](https://gpuopen.com/learn/concurrent-execution-asynchronous-queues/) and [NVIDIA](https://www.khronos.org/assets/uploads/developers/library/2016-vulkan-devday-uk/9-Asynchonous-compute.pdf)
+
+An application submits work to a `VkQueue`, normally in the form of `VkCommandBuffer` objects or [sparse bindings](./sparse_resources.md).
+
+Command buffers submitted to a `VkQueue` start in order, but are allowed to proceed independently after that and complete out of order.
+
+Command buffers submitted to different queues are unordered relative to each other unless you explicitly synchronize them with a `VkSemaphore`.
+
+You can only submit work to a `VkQueue` from one thread at a time, but different threads can submit work to different a `VkQueue` simultaneously.
+
+The concept of a `vkQueue` is implementation-defined. Some hardware have multiple hardware queues and submitting work to multiple `vkQueue` will proceed independently and concurrently in the hardware. Some implementation will do some scheduling at a kernel driver level before being submitted to the hardware. There is no current way in Vulkan to expose the exact details how each `vkQueue` is mapped to the implementation.
```

Comment:
I don't think the "concept" is implementation defined. I think you're trying to say that it's implementation-defined how a queue is mapped to the underlying hardware (sort of what the last sentence of the paragraph is saying.

Also, the grammar in the second sentence is a bit broken. I think "will proceed" points back to the "submitting" in the sentence, but I suspect that what you're trying to express is that the submitted work proceeds in parallel. It would also be good to avoid using "hardware" three times in the same sentence.

Finally,
>Some implementation will do some scheduling at a kernel driver level before being submitted to the hardware.
Maybe:
=> Some implementations will do scheduling at a kernel driver level before submitting work to the hardware.

Reply:
Agree and reworded the sentence to flow better, not repeat, and get rid of "concept"